ui/UI_GUIDELINES.md

```diff
@@ -0,0 +1,26 @@
+# UI principles and guidelines
```

Comment:
The content is more about architectural principles and technologies/languages/libraries used. The title is quite misleading, maybe we can find a better one.